src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -607,15 +615,19 @@ private void validateRules() {
     }
   }
 
+  private String getNumberOfWinnersRaw() {
+    return rawConfig.rules.numberOfWinners;
+  }
+
   // function: getNumberWinners
   // purpose: how many winners for this contest
   // returns: number of winners
   Integer getNumberOfWinners() {
-    return rawConfig.rules.numberOfWinners;
+    return Integer.parseInt(rawConfig.rules.numberOfWinners);
```

Comment:
So now we're trusting that a callsite will never call a getter like this unless the config has already been validated... right?

Reply:
Yes, that's correct. I checked carefully, but if you want to add some further warnings or checks somewhere, we can file an issue.